RAP/rap-statistics.qmd

```diff
@@ -382,86 +485,84 @@ By following [our guidance](#Version_controlled_final_code_scripts) in saving ve
 
```

Comment:
version controlled final code scripts link goes to the top of the RAP for stats page for me

Reply:
needed hyphens instead of underscores - fixed!